src/Function.idr

```diff
@@ -60,8 +60,8 @@ data FunctionEquality: {A, B: Setoid} -> (f, g: Function A B) -> Type where
                     -> FunctionEquality f g
 
 ||| Proof that defined equality of extensional functions is indeed equality
-functionEqualityIsEquality: {A, B: Setoid} -> IsEquality (FunctionEquality {A} {B})
-functionEqualityIsEquality {A} {B} with (B)
+functionEqualityIsEquality: {a, b: Setoid} -> IsEquality (FunctionEquality {A = a} {B = b})
```

Comment:
Pattern variables can't start with capital letters since Idris v1.2 (my bad, I thought since v1.3), so I changed all the names used in this way. Also [there's a switch](https://github.com/idris-lang/Idris-dev/blob/master/CHANGELOG.md#tool-updates) that allows using capital letters, but I'm afraid that sooner or later it will be deleted.

Reply:
Ok, didn't know about it